doc/api/fs.md

```diff
@@ -295,10 +295,14 @@ argument to `fs.createReadStream()`. If `path` is passed as a string, then
 ## Class: fs.Stats
 <!-- YAML
 added: v0.1.21
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/13173
+    description: added times as Numbers.
```

Comment:
Can you capitalize `Added` as the beginning of a sentence, and lowercase `numbers`?